manim/scene/scene_file_writer.py

```diff
@@ -106,6 +100,32 @@ def init_output_directories(self, scene_name):
                 )
             )
 
+    def init_save_image(self, scene_name):
+        """Save images to the output file directory or the path of the scene. 
+        It does this after automatically detecting no animations are present and 
+        making the output an image instead of a video. 
+        """
+        if config["output_file"]:
+            default_name = config.get_dir("output_file")
+        else:
+            default_name = Path(scene_name)
+
+        if config["input_file"]:
+            module_name = config.get_dir("input_file").stem
+        else:
+            module_name = ""
+
+        if config["media_dir"]:
+            image_dir = guarantee_existence(
+                config.get_dir("images_dir", module_name=module_name)
+            )
+        self.image_file_path = os.path.join(
+            image_dir, add_extension_if_not_present(default_name, ".png")
+        )
+        logger.info(
+                "Rendered 0 animations"
+            )
```

Comment:
Nitpick : This is not necessary, as there is already such log at l. 110 of scene.py

Reply:
+1, please remove the duplicated log message here. I do think, however, that it would be helpful to add something like

```py
logger.info("This scene contains no animations, the output has been saved as an image.")
```

in `scene.py` after logging "Played 0 animations".